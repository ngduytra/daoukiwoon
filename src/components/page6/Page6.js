import React from 'react';
import './page6.styles.scss'
import Computer from '../../assets/img/Computer.png'

const Page6 = () => {
    return (
        <div id='index-6'>
            <div className="content">
                <img src={Computer} />
                <div className='right'>
                    <div className='title'>CUSTOMER</div>
                    <div className='subtitle'>Customer Request Management</div>
                    <div className='button'>OQUFIE</div>
                </div>
            </div>
        </div>
    )
}

export default Page6;