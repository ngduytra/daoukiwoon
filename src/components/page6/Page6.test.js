import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import React from 'react';
import Page6 from './Page6';

configure({adapter: new Adapter()});

it('expect to render Page6 component', ()=>{
    expect(shallow(<Page6/>)).toMatchSnapshot();
})