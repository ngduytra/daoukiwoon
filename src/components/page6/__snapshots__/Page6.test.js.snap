// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Page6 component 1`] = `ShallowWrapper {}`;
